lib/bonny/crd.ex

```diff
@@ -17,20 +17,29 @@ defmodule Bonny.CRD do
           version: String.t()
         }
 
+  @type columns_t :: %{
+          name: String.t(),
+          type: String.t(),
+          description: String.t(),
+          JSONPath: String.t()
+        }
+
   @typedoc "CRD Spec"
   @type t :: %__MODULE__{
           scope: :namespaced | :cluster,
           group: String.t(),
           names: names_t,
-          version: String.t()
+          version: String.t(),
+          additionalPrinterColumns: list(columns_t)
```

Comment:
Casing is fine as is since this gets serialized to k8s

Reply:
I didn't understand :/